packages/client/src/__tests__/integration/errors/missing-engine/library.test.ts

```diff
@@ -41,30 +41,19 @@ test('missing-engine: library', async () => {
       32 
       33 await expect(async () => {
     → 34   await prisma.user.findMany(
-    Query engine library for current platform "TEST_PLATFORM" could not be found.
-    You incorrectly pinned it to TEST_PLATFORM
+    Prisma Client could not locate the Query Engine for runtime "TEST_PLATFORM".
 
-    This probably happens, because you built Prisma Client on a different platform.
-    (Prisma Client looked in "/client/src/__tests__/integration/errors/missing-engine/node_modules/@prisma/client/runtime/libquery_engine-TEST_PLATFORM.LIBRARY_TYPE.node")
+    This is likely caused by tooling that has not copied "libquery_engine-TEST_PLATFORM.LIBRARY_TYPE.node" to the deployment.
+    Please try to make sure that "libquery_engine-TEST_PLATFORM.LIBRARY_TYPE.node" is copied to "src/__tests__/integration/errors/missing-engine/node_modules/.prisma/client".
```

Comment:
Note: For me, while building a Dockerfile I happened to have that issue and the fix was to run `prisma generate`.

Reply:
Good point, will add this to the message.